docs/release-notes.txt

```diff
@@ -8,6 +8,128 @@
  Version 0.13
 **************
 
+Version 0.13.8
+==============
+
+**Release Date:** November 17, 2020
+
+**New Features**
+
+-  Deployment: When using ``det-deploy aws``, you can now specify the 
+   subnet that the agents should be launched into (if you are using the 
+   simple deployment type). This allows users to explicitly pick a
+   subnet/availability zone that has GPU instances.
+
+-  Security: Allow the agent to use different addresses for master 
+   certificate verification and connecting to the master.
+
+**Improvements**
+
+-  **Breaking Change:** API: Perform salting and hashing on server-side for
+   the password change endpoint. This makes this endpoint consistent with
+   the new login endpoint described at
+   https://docs.determined.ai/latest/rest-api/ .
+
+-  **Breaking Change:** Logging: Start using Fluent Bit for handling trial 
+   logs internally. The agent machines now need to have access to the 
+   ``fluent/fluent-bit:1.6`` Docker image. If the Determined agent machines 
+   are able to connect to Docker Hub, they will pull it automatically and 
+   no changes are required; if not, the image must be manually made 
+   available beforehand. The Determined agent accepts a 
+   ``--fluent-logging-image`` option to specify an alternate name for the 
+   image.
+
+-  Agent: Support configurable slot types for agents. Previously, 
+   Determined only supported auto-detecting the slot type for agents. If  
+   Determined did not detect any GPUs, the agents would fall back to 
+   mapping one slot to all the CPUs.  With this change, this behavior can 
+   be configured to one of ``auto``, ``gpu``, and ``none`` in the field 
+   ``slot_type`` of the agent configuration ``agent.yaml``. Dynamic agents 
+   having GPUs will be configured to ``gpu`` while those agents having no 
+   GPUs will be configured to ``none``. For static agents this field 
+   defaults to ``auto``. See 
+   :ref:`Agent configuration <agent-configuration>` for details.
+
+-  API: Add ``self.context.wrap_optimizer()`` to the Determined
+   TFKerasTrial API. Please see :ref:`tf-keras-trial` for detail.
+
+-  API: Add support for models that subclass ``tf.keras.Model`` when 
+   using the Determined TFKerasTrial API.
+
+-  API: Add tf.keras DCGAN example that subclasses ``tf.keras.Model``.
+
+-  API: Add ``self.context.configure_fit()`` to the Determined
```

Comment:
What does this actually enable? Can we give an example of what this is useful for? @rb-determined-ai

Reply:
how about this:
```rest
-  API: Add ``self.context.configure_fit()`` to the Determined
   TFKerasTrial API. Many parameters which would be passed to
   ``model.fit()``, such as ``class_weight``, ``verbose``, or
   ``workers``, can now be passed to ``configure_fit()`` and will be
   honored by ``TFKerasTrial``.  Please see :ref:`tf-keras-trial` for
   detail.
```